vendor/golang.org/x/crypto/ocsp/ocsp.go

```diff
@@ -41,6 +41,7 @@ const (
 	// https://tools.ietf.org/html/rfc6960#section-4.2.1
 	SignatureRequired ResponseStatus = 5
 	Unauthorized      ResponseStatus = 6
+	Deprecated        ResponseStatus = 7
```

Comment:
We do not allow directly editing files under `vendor/`. Those are automatically generated by `go mod vendor`. If you need to change a file under `vendor/`, first you need to make that change to the upstream repository (in this case https://github.com/golang/go/), get it merged, and then run `go get -u` in this package.

However, changes to the upstream repository in this case are not needed. For the OCSP response body, you should return the `Unauthorized` pregenerated response. The OCSP protocol doesn't have a notion of "deprecated", but "unauthorized" fits: https://datatracker.ietf.org/doc/html/rfc6960#section-2.3

>    The response "unauthorized" is returned in cases where the client is
>   not authorized to make this query to this server or the server is not
>   capable of responding authoritatively